### PR TITLE
Sourcr refactor. Abstracted common functionality and merged two fairly l...

### PR DESCRIPTION
**Pull request by krishnanm86**
...are functions into filebasedsvsbynamespace in order to sace redundant code

### CODE REVIEW

**Comment by danielmewes** [comment]
Thanks for this one too @krishnanm86.
Also added a comment. Looks good apart from that.

File: src/clustering/administration/main/file_based_svs_by_namespace.cc [hunk 1]
```diff
@@ -45,13 +45,14 @@ std::string hash_shard_perfmon_name(int hash_shard_number) {
     return strprintf("shard_%d", hash_shard_number);
 }
 
-void do_construct_existing_store(
+void do_store(
```
Comment (danielmewes) [inline src/clustering/administration/main/file_based_svs_by_namespace.cc]: Can you rename this from `do_store()` to `do_construct_store()`?

**Comment by krishnanm86** [comment]
Done. Hopefully it can be merged now.

**Comment by danielmewes** [comment]
Thanks @krishnanm86 !

**Pull request merged**